docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- The FrSky R9M can be flashed using your EdgeTX or OpenTX radio.
+- No 3D printer is required, but transmit power should be limited to less than 500 mW unless you add a fan.
+- The Bayck 915M Nano Pro ELRS receiver can be [flashed](ELRS_RECEIVERS.md#flashing) with mLRS firmware without removing it from your vehicle by using ArduPilot serial passthrough.
+- If you already have a R9MX or R9MM receiver, they are also compatible, but have lower power and are more difficult to [flash](FRSKY_R9.md#flash-r9-receivers-with-elrs-bootloader).
+- Unfortunately, the R9M has inverted serial port signals; if you choose this option you will probably also want to purchase an [ESP32 module](FRSKY_R9.md#esp32-wireless-bridge) such as the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) to use as a wireless bridge and handle the inverted serial port.
+
+## 2. Assemble your Tx module if required and install antennas on both the receiver and Tx module
+
+If you are using one of the MatekSys Tx module kits, print or purchase a JR bay case from the STL download available from [MatekSys](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-1).
+
+If you will use the MatekSys HC04 module to connect your ground station via Bluetooth, set all three [dip switches](https://MATEKSYS.md#tx-module-hc-04-bluetooth-notes) to the on position before assembling your Tx module.
+
+If you are using the [R9M Tx module](FRSKY_R9.md#r9m-tx-module):
+- Set dip switch 1 to the On position to configure it for serial data
+- Install the [Arduino IDE](https://www.arduino.cc/en/software) and use it to build and flash [firmware](WIRELESS_BRIDGE.md) on your ESP32 module.  Be sure to select the R9M variant for your module when you edit the mlrs-wireless-bridge.ino.
+- Connect your [ESP32 wireless bridge](https://FRSKY_R9.md#esp32-wireless-bridge) module to the serial and power pins of your R9M.
+
+When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is, unfortunately, possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal.
+
+## 3. Connect your receiver to your flight controller
+
+For the 1 watt MatekSys [mLRS receivers](https://www.mateksys.com/?portfolio=mr24-30#tab-id-2):
+- connect the receiver Rx1 to a FC UART Tx.
+- connect the receiver Tx1 to the corresponding FC UART Rx.
+- Connect the receiver V pin to a 5 volt output on your FC.
+- Connect the receiver G pin to a ground pad on your FC.
+- If you will use CRSF for RC channels (recommended), connect the receiver TX2 to a different UART Rx on your FC.
+
+For most ELRS receivers, the [connections](ELRS_RECEIVERS.md#connections) are similar to those above except CRSF is not available.
+
+Connect your ground station to your ArduPilot flight controller via USB and configure the serial port (x) you wired to your receiver serial port.  Don't forget to press "write parameters" if your ground station requires this step.
```

Comment:
is there a chance for confusion which USB?
maybe ".. via the flight controller's USB ..."

Reply:
I didn't think so since it's USB on both ends, but if you are asking, there might be.  How about "Directly connect your ground station to your ArduPilot flight controller via a USB cable"  Of course, it doesn't really matter how they connect.  They could use another serial port or even another telemetry radio.  USB is just the easy way everyone is likely to have available.